src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -1,5 +1,41 @@
 import styled from "styled-components";
 export const HandbookWrapper = styled.div`
+
+
+.highlight{
+
+  font-weight: 600;
+}
+.content{
+  width:100%;
+  padding-bottom: 2rem;
+ul > li {
+    color:#00d3a9;
```

Comment:
Can you use colors from the theme instead of hardcoding everywhere

Reply:
@Tanuj22 the colors which are not in the theme I'm leaving them hardcoded, is that okay?

Reply:
We shouldn't be using any colors that are not in the theme though :thinking:

Reply:
Which color is being used that is not in the theme?

Reply:
These are there from before😅, I havent added any separate colors

Reply:
#f1f1f1, #929496 these are colors added for hover effect

Reply:
these can probably be replaces by `lighter gray` which a color in our theme

Reply:
Okay. I can't locate the colors though. If replacing works go ahead